src/main/resources/view/graphEditView.fxml

```diff
@@ -1,21 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.scene.layout.*?>
-
-<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.HBox?>
 <HBox xmlns="http://javafx.com/javafx"
-        xmlns:fx="http://javafx.com/fxml"
-        fx:controller="com.example.controller.GraphEditController">
+      xmlns:fx="http://javafx.com/fxml"
+      fx:controller="com.example.controller.GraphEditController">
 
-    <Button text = "Vertex" fx:id="vertexButton"/>
-    <Button text = "Edge" fx:id="edgeButton"/>
-    <Button text = "Delete" fx:id="deleteButton"/>
-    <Button text = "None" fx:id="noneButton"/>
-    <Button text = "Simulate" fx:id="simulateButton"/>
-    <Button text="Start" fx:id="startButton" />
-    <Button text="Next Step" fx:id="nextStepButton" />
-    <Button text="Live" fx:id="liveButton" />
-    <Button text="Instant finish" fx:id="instantFinishButton" />
-    <Button text="Pause" fx:id="pauseButton" />
-    <Button text = "Draw" fx:id="drawButton"/>
+    <Button text="Vertex" fx:id="vertexButton"/>
+    <Button text="Edge" fx:id="edgeButton"/>
+    <Button text="Delete" fx:id="deleteButton"/>
+    <Button text="None" fx:id="noneButton"/>
+    <ToggleButton text="Simulate" fx:id="simulateButton"/>
```

Comment:
Simulate button appears both at the beginning and at the end depending on mode we use. I'd be consistent and move it.

Reply:
Fixed